android-components/components/compose/awesomebar/src/main/java/mozilla/components/compose/browser/awesomebar/internal/Suggestions.kt

```diff
@@ -37,11 +37,9 @@ internal fun Suggestions(
         modifier = Modifier.testTag("mozac.awesomebar.suggestions"),
     ) {
         suggestions.forEach { (group, suggestions) ->
-            if (suggestions.isNotEmpty()) {
-                group.title?.let { title ->
-                    item(group.id) {
-                        SuggestionGroup(title, colors)
-                    }
+            if (suggestions.isNotEmpty() && !group.title.isNullOrEmpty()) {
```

Comment:
To avoid using `!!` maybe this is better?
```suggestion
            val title = group.title
            if (suggestions.isNotEmpty() && !title.isNullOrEmpty()) {
                item(group.id) {
                    SuggestionGroup(title, colors)
                }
            }
```

Reply:
Agreed, we should try to avoid `!!`